mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -0,0 +1,129 @@
+CLASS net/minecraft/class_43 net/minecraft/world/chunk/Chunk
+	FIELD field_954 blocks [B
+	FIELD field_956 world Lnet/minecraft/class_18;
+	FIELD field_957 meta Lnet/minecraft/class_257;
+	FIELD field_958 skyLight Lnet/minecraft/class_257;
+	FIELD field_959 blockLight Lnet/minecraft/class_257;
+	FIELD field_960 heightmap [B
+	FIELD field_961 minHeightmapValue I
+	FIELD field_962 x I
+	FIELD field_963 z I
+	FIELD field_964 blockEntities Ljava/util/Map;
+	FIELD field_965 entities [Ljava/util/List;
+	METHOD <init> (Lnet/minecraft/class_18;II)V
+		ARG 1 world
+		ARG 2 x
+		ARG 3 z
+	METHOD <init> (Lnet/minecraft/class_18;[BII)V
+		ARG 1 world
+		ARG 2 blocks
+		ARG 3 x
+		ARG 4 z
+	METHOD method_858 (II)Z
```

Comment:
could this be net.minecraft.util.math.ChunkPos.equals in modern ?
![obrazek](https://github.com/calmilamsy/biny-mappings/assets/24621316/c7186c03-fb39-4d62-95b3-9a8f7f1f8678)
so maybe something like chunkPosEquals ?

Reply:
I'd say chunkPosEquals too. It's not behaving the same as the java.lang.Object:Equals method, so it shouldn't share the name.